steps/Login.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+using NUnit.Framework;
+
+namespace MarsQA_1.Feature
+{
+    [Binding]
+    class Login
+    {
+     
+    
+        [Given(@"I navigate to application")]
+        public void GivenINavigateToApplication()
+        {
+
+            Pages.SignIn.Login();

```

Comment:
For this task, you need to use Specflow to define the cases and also data

Additionally, it's good to pass the username and password as parameter here